tests/components/tplink/snapshots/test_sensor.ambr

```diff
@@ -146,7 +146,7 @@
     'supported_features': 0,
     'translation_key': 'battery_level',
     'unique_id': '123456789ABCDEFGH_battery_level',
-    'unit_of_measurement': '%',
```

Comment:
We miss the units & suggested display precisions as their values are first initialized after `async_add_to_hass` has been called & this is not currently done.